src/test/java/io/github/classgraph/utils/JarUtilsTest.java

```diff
@@ -0,0 +1,52 @@
+package io.github.classgraph.utils;
+
+import io.github.classgraph.utils.JarUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class JarUtilsTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+  /* testedClasses: JarUtils.java */
+
+  @Test
```

Comment:
Do they have full coverage on the rest of this method? Is this test covering an edge case that they didn't think of?

Reply:
It is unfortunately the only test we got for this method. Couldn't detect existing coverage, but they don't appear to have any existing tests for this class.